modules/core/quick/tests/src/Functional/QuickFormTest.php

```diff
@@ -91,8 +91,8 @@ public function testQuickForms() {
     $this->assertSession()->statusCodeEquals(200);
     $this->assertSession()->responseContains('value="100"');
 
-    // Go to the test configuration form and confirm that access is denied.
-    $this->drupalGet('quick/configurable_test/configure');
+    // Go to the test edit form and confirm that access is denied.
+    $this->drupalGet('setup/quick/configurable_test/edit');
     $this->assertSession()->statusCodeEquals(403);
```

Comment:
Looks like this test is failing because the `setup/quick/configurable_test/edit` path is returning a 404.

Reply:
Aha. This is because the quick form config entity had not yet been created in these tests. Later on it does create an config entity and do similar tests so I moved this logic there.